### ISSUE TITLE
WCAG: Multiple select delete button is missing label

### ISSUE THREAD

**Issue by bjosttveit** [digdir/designsystem#280]
### Description of the bug

Wave gives an error that the `X` (delete) button is missing a label. This is for both individual items and the delete all button.

<img width="624" alt="image" src="https://user-images.githubusercontent.com/47412359/228792223-83a5c67d-90a8-4cf0-b638-f8b18720fd73.png">

### Steps To Reproduce

.

**Comment by TomasEng** [comment]
Closed because it is possible to set these labels with props.
